drivers/modem/Kconfig.gsm

```diff
@@ -78,6 +78,19 @@ config MODEM_GSM_MANUAL_MCCMNO
 	  the network provider and may need to be changed if auto is not
 	  selected.
 
+config MODEM_GSM_RSSI_POLLING_PERIOD
+	int "Configure RSSI polling period"
```

Comment:
Please add info about the resolution either here in the prompt, or in the help text. Otherwise it is impossible for the casual reader to understand what the value means. I would recommend adding `(in seconds)` to the prompt.